runner/total_stats_coordinator.rb

```diff
@@ -0,0 +1,65 @@
+require_relative '../config/init'
+require_relative '../app/models/total_stats'
+require 'pg'
+
+module PodStats
+
+  class TotalStatsCoordinator
+    attr_accessor :connection
+
+    def get_all_targets type
+      type_id = PRODUCT_TYPE_UTI[type]
+
+      query = <<-eos
```

Comment:
use `SQL` instead of `eos`, so some text editors will be able to highlight it correctly